apps/nmap/cli/mode/discoverynmap.pm

```diff
@@ -18,13 +18,12 @@
 # limitations under the License.
 #
 
-package os::linux::local::mode::discoverynmap;
+package apps::nmap::cli::mode::discoverynmap;
```

Comment:
Shouldn't it be just "discovery" with the mode being named "discovery.pm" ? What do you think ?

Reply:
You're obviously right!
Done.